api/src/shop/stripe_subscriptions.py

```diff
@@ -98,13 +98,13 @@ def calc_start_ts(current_end_date: date, now: datetime) -> datetime:
 def calc_subscription_start_time(
     member_id: int,
     subscription_type: SubscriptionType,
-    earliest_start_at: Optional[datetime],
+    earliest_start_at: Optional[datetime] = None,
```

Comment:
Why make this parameter have a default value? The default value is not used anywhere.

Reply:
Fixed